content/docs/pipelines/sdk/python-based-visualizations.md

```diff
@@ -58,9 +58,29 @@ arguments.
 6. Click **Generate Visualization**.
 7. View generated visualization by scrolling down.
 
+## Using TFX visualizations
+
```

Comment:
We should add some context here. When I first saw the section, I was confused because the page said there are **two** types of visualizations (predefined and custom) but now there seems to be a third kind. I didn't know how TFX visualizations fit in. 

Suggested text:

```
TFX visualizations are one of the predefined visualization types available. The following video and instructions show you how to select the TFX visualization type (**TFDV**), generate the visualization, and view the results.
```

Reply:
I am thinking of making two subsections under ## Using predefined visualizations. One is ### Predefined matrix visualization (which is the existing example), and the other is ### Predefined TFX visualization (which is the tfx example I am adding)

Reply:
That looks great!